Mathlib/Order/Interval.lean

```diff
@@ -673,12 +676,17 @@ section CompleteLattice
 
 variable [CompleteLattice α]
 
+-- Porting note: (TODO) The API around `OrderDual` seems very sketchy to me.
+-- I used this helper lemma in the proof below but, somehow it seems that's
+-- just an indication that the API does not work. In mathlib3, it would automatically do stuff
+-- (this lemma should probably be removed again, once things work)
+theorem OrderDual.le_def (x y : αᵒᵈ) [h : LE α] : (OrderDual.le α).le x y ↔ h.le y x := by rfl
```

Comment:
As noted on Zulip, we definitely don't want this lemma

Reply:
done